corehq/apps/locations/templates/locations/location_types.html

```diff
@@ -157,15 +157,17 @@ <h2>{% trans "Organization Levels" %}</h2>
                                           {% endblocktrans %}">
                                     </span>
               </th>
-              <th>
-                {% trans "View Child Data to Level" %}
-                <span class="hq-help-template"
-                      data-title="{% trans "View Child Data to Level" %}"
-                      data-content="{% blocktrans %}
-                          <p>Only include cases that show up in the user's restore file down to this level (inclusive).</p>
-                      {% endblocktrans %}">
-                </span>
-              </th>
+              {% if request|toggle_enabled:"USH_RESTORE_FILE_LOCATION_CASE_SYNC_RESTRICTION" %}
+                <th data-bind="visible: advanced_mode">
```

Comment:
Why do you need both the feature flag and another boolean to display this section? Wouldn't one or the other be enough? (Same question applies for below)

Reply:
The `advanced_mode` flag is a checkbox that users enable to show more columns:
![image](https://github.com/dimagi/commcare-hq/assets/2367539/3ce80fe8-69c0-4645-b959-859db256cb6e)

Reply:
gotcha, thanks for explaining yall!